content/guides/http/context.md

```diff
@@ -43,7 +43,7 @@ class HomeController {
 }
 ```
 
-## Differences from the Express `req` and `res` objects?
+## Are there any relation to `req` and `res` objects from the Express?
```

Comment:
I am not sure about this phrasing. `the express` should be only `Express`.

Reply:
Agree! @nazarepiedady Can you please create a PR to remove `the`?